### PR TITLE
Eui 389 log inand log out automation

### PR DESCRIPTION
**Pull request by kommurunagendra**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: test/e2e/config/conf.js [hunk 1]
```diff
@@ -6,6 +6,17 @@ chai.use(chaiAsPromised);
 
 const argv = minimist(process.argv.slice(2));
 
+//const specFilesFilter = ['../features/**/*.feature'];
```
Comment (fazanki) [inline test/e2e/config/conf.js]: can u remove this or make a comment why its commented out

File: test/e2e/config/conf.js [hunk 2]
```diff
@@ -36,13 +47,21 @@ const config = {
   framework: 'custom',
   frameworkPath: require.resolve('protractor-cucumber-framework'),
   specs: ['../features/**/*.feature'],
-  baseUrl: process.env.TEST_URL || 'http://localhost:2500',
+  // specs: [
```
Comment (fazanki) [inline test/e2e/config/conf.js]: please remove or comment why is commented out

File: test/e2e/config/conf.js [hunk 3]
```diff
@@ -36,13 +47,21 @@ const config = {
...
+  //   '../features/**/caseFile.feature',
+  //   '../features/**/login.feature',
+  //   '../features/**/makeDecision.feature',
+  //   '../features/**/parties.feature',
+  //   '../features/**/questions.feature',
+  //   '../features/**/recentEvents.feature',
+  // ],
+  baseUrl: process.env.TEST_URL || 'http://localhost:3000',
```
Comment (fazanki) [inline test/e2e/config/conf.js]: please add this to config file (http:localhost.... )

File: test/e2e/config/conf.js (hunk 1 above)
Comment (kommurunagendra) [reply test/e2e/config/conf.js]: feature improvements

File: test/e2e/config/conf.js (hunk 2 above)
Comment (kommurunagendra) [reply test/e2e/config/conf.js]: feature improvements

File: test/e2e/config/conf.js (hunk 3 above)
Comment (kommurunagendra) [reply test/e2e/config/conf.js]: feature improvements

**Pull request merged**